doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,127 @@
+r"""
+==========================================
+Interpolate images with thin-plate splines
+==========================================
+
+Thin-plate splines (TPS) refer to a method for interpolating data [1]_.
+In an image context, we use a 2D generalization of TPS, i.e., a two-dimensional
+simulation of a 1D cubic spline, which is the solution of the biharmonic equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
+Given corresponding source and target control points, TPS is used to transform a space.
+In this case our space is a 2D image.
+
+It should be noted that the commonly used in image processing
+Affine Transformation [4]_ can be understood as special variant of TPS.
+
+This script shows how to use the Thine Plate Spine Transformation for
+image warping.
+
+For further information on TPS Transformation, see:
+
+.. [1] Wikipedia, Thin plate spline
+       https://en.wikipedia.org/wiki/Thin_plate_spline
+
+.. [2] Weisstein, Eric W. "Biharmonic Equation." From MathWorld--A Wolfram Web Resource
+       https://mathworld.wolfram.com/BiharmonicEquation.html
+
+.. [3] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
+       decomposition of deformations." IEEE Transactions on pattern analysis and
+       machine intelligence 11.6 (1989): 567–585.
+
+.. [4] Wikipedia, Affine transformation
+       https://en.wikipedia.org/wiki/Affine_transformation#Image_transformation
+
+
+Image Deformation
+=================
+
+In this example we will see how to use thin plates spline interpolation in the
+context of deforming an astronaut image. To deform the image, the displacement of every
+pixel is needed. In our image we define 6 source and destination points labelled 1-6: 1-4 in the image
+corners, 5 near the smile corner and 6 in an eye. At the corners, there are no
+displacements. The smile corner moved upward and the eye position moved down.
+
+Thin Plate Splines provides a very handy interpolator for image deformation.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import skimage as ski
+
+astronaut = ski.data.astronaut()
+
+src = np.array([[50, 50], [400, 50], [50, 400], [400, 400], [240, 150], [200, 100]])
+dst = np.array([[50, 50], [400, 50], [50, 400], [400, 400], [276, 100], [230, 100]])
+
+# Fit the thin plate spline from output to input
+warped_img = ski.transform.tps_warp(astronaut, src[:, ::-1], dst[:, ::-1], grid_scaling=1)
+
+
+fig, axs = plt.subplots(1, 2, figsize=(16, 8))
+
+labels = ['1', '2', '3', '4', '5', '6']  # Adjust the number of labels to match the number of points
+
+axs[0].imshow(astronaut[..., ::-1], origin='upper', cmap='gray')
```

Comment:
```suggestion
axs[0].imshow(astronaut, origin='upper', cmap='gray')
```
I think the `::-1` in the last channel axis is responsible for the [wrong coloring in the example](https://output.circle-artifacts.com/output/job/5f8e0769-a583-4b85-81f3-914b13a933b5/artifacts/0/doc/build/html/auto_examples/transform/plot_tps_deformation.html#sphx-glr-auto-examples-transform-plot-tps-deformation-py).

Same below. Or is there another reason for this?